Documentation/Blazorise.Docs/Pages/Docs/Extensions/QRCode/QRCodePage.razor

```diff
@@ -78,6 +93,14 @@
     To optimize your web for SEO, define the <Code>Alt</Code> parameter to describe the QR image content, eg. <Code>Alt="QRCode image"</Code>.
 </DocsPageParagraph>
 
+<Heading Size="HeadingSize.Is3">
+    Increase Error Level for Images
```

Comment:
"Error Level" seems wrong, **ECC**, stands for **Error Correction Level**, and  I think that's better for interpretation.